### PR TITLE
Add MaxBIOSVersion option to Generic

### PR DESCRIPTION
**Pull request by khronokernel**
Reference: https://github.com/acidanthera/bugtracker/issues/1427

### CODE REVIEW

**Comment by vit9696** [comment]
Maybe just make it a bool? MaxBIOSVersion or something?

File: Platform/OpenCore/OpenCorePlatform.c [hunk 1]
```diff
@@ -373,6 +373,12 @@ OcPlatformUpdateSmbios (
         DEBUG ((DEBUG_WARN, "OC: Invalid SMBIOS system memory status %a\n", SystemMemoryStatus));
       }
     }
+	//
```
Comment (PMheart) [inline Platform/OpenCore/OpenCorePlatform.c:376]: Please fix indentation here.

File: Include/Acidanthera/Library/OcConfigurationLib.h [hunk 2]
```diff
@@ -448,6 +448,7 @@ typedef enum {
   _(OC_STRING                   , SystemUuid         ,     , OC_STRING_CONSTR ("", _, __)                 , OC_DESTR (OC_STRING) ) \
   _(OC_STRING                   , Mlb                ,     , OC_STRING_CONSTR ("OPENCORE_MLB_SN11", _, __), OC_DESTR (OC_STRING) ) \
   _(OC_STRING                   , SystemMemoryStatus ,     , OC_STRING_CONSTR ("Auto", _, __)             , OC_DESTR (OC_STRING) ) \
+  _(OC_STRING                   , BIOSVersion        ,     , OC_STRING_CONSTR ("", _, __)                 , OC_DESTR (OC_STRING) ) \
```
Comment (PMheart) [inline Include/Acidanthera/Library/OcConfigurationLib.h:451]: Should be
`_(OC_STRING                   , BIOSVersion        ,     , OC_STRING_CONSTR ("Auto", _, __)                 , OC_DESTR (OC_STRING) ) \`

File: Platform/OpenCore/OpenCorePlatform.c [hunk 3]
```diff
@@ -373,6 +373,12 @@ OcPlatformUpdateSmbios (
...
+	// Permit overriding BIOSVersion.
+	//
+    if (OC_BLOB_GET (&Config->PlatformInfo.Generic.BIOSVersion)[0] != '\0') {
```
Comment (PMheart) [inline Platform/OpenCore/OpenCorePlatform.c:379]: Make sure that it is not `Auto` here.

**Pull request merged**